src/utils/index.js

```diff
@@ -14,7 +14,20 @@ export function getDataTransferItems(event) {
     } else if (dt.items && dt.items.length) {
       // During the drag even the dataTransfer.files is null
       // but Chrome implements some drag store, which is accesible via dataTransfer.items
-      dataTransferItemsList = dt.items
+      // NOTE: Allow only files and retun the items as a list of File,
+      // see https://developer.mozilla.org/en-US/docs/Web/API/DataTransferItem for details
+
+      const items = []
+
+      // eslint-disable-next-line no-plusplus
+      for (let i = 0; i < dt.items.length; i++) {
```

Comment:
I don't like imperative `for` loops TBH. Let's use `Array.prototype.filter` here